src/registrar/views/domain.py

```diff
@@ -2,7 +2,7 @@
 
 Authorization is handled by the `DomainPermissionView`. To ensure that only
 authorized users can see information on a domain, every view here should
-inherit from `DomainPermissionView` (or DomainInvitationPermissionDeleteView).
+inherit from `DomainPermissionView`.
```

Comment:
Is this a change in our coding practices? What's the justification for no longer inheriting from DomainInvitationPermission{Delete/Cancel}View?

Reply:
I just removed it from this text since DomainInvitationPermissionDeleteView no longer exists. I will add DomainInvitationPermissionCancelView! This is a great catch.